pushnotifications-integration-tests/src/test/java/com/example/pushnotificationsintegrationtests/ExampleUnitTest.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.pushnotificationsintegrationtests
+
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
```

Comment:
This could be removed since it's example test case.

Reply:
Right, I think I've removed it in another branch.